### PR TITLE
Fix text align

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: stories/browser/src/UICellDebot/UICellDebot.tsx [hunk 1]
```diff
@@ -30,26 +30,26 @@ function UICellDebotImpl({ image, title, caption, onPress, loading, testID }: UI
...
-                            {caption}
-                        </UILabel>
-                    ) : null}
-                </View>
+                {title ? (
+                    <UILabel
+                        role={TypographyVariants.NarrowActionFootnote}
+                        color={ColorVariants.TextPrimary}
+                        numberOfLines={caption ? 1 : 2}
+                        style={styles.textContainer}
```
Comment (sertony) [inline stories/browser/src/UICellDebot/UICellDebot.tsx:38]: UILabel does not have `style` prop (and can be used only for internal purposes).
@savelichalex any ideas?

Comment (savelichalex) [reply stories/browser/src/UICellDebot/UICellDebot.tsx:38]: Let's say it's still internal, as it's in @story :)

File: stories/browser/src/UICellDebot/UICellDebot.tsx [hunk 2]
```diff
@@ -30,26 +30,26 @@ function UICellDebotImpl({ image, title, caption, onPress, loading, testID }: UI
...
+                    >
+                        {title}
+                    </UILabel>
+                ) : null}
+                {caption ? (
+                    <UILabel
+                        role={TypographyVariants.NarrowParagraphFootnote}
+                        color={ColorVariants.TextSecondary}
+                        numberOfLines={title ? 1 : 2}
+                        style={styles.textContainers}
```
Comment (savelichalex) [inline stories/browser/src/UICellDebot/UICellDebot.tsx]: There is no `textContainers` is it a miss-type?

File: stories/browser/src/UICellDebot/UICellDebot.tsx (hunk 1 above)
Comment (sertony) [reply stories/browser/src/UICellDebot/UICellDebot.tsx:38]: What about TypeScript?)

Comment (savelichalex) [reply stories/browser/src/UICellDebot/UICellDebot.tsx:38]: @BigRoma does it pass TS checks?

**Review by sertony — APPROVED** [review]

**Pull request merged**